tests/autobahn/run-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+rm -rf $PWD/reports
+mkdir $PWD/reports
+
+USER_ID=$(id -u) docker-compose -p aiohttp-autobahn -f $PWD/images-for-compose.yml build --parallel
+
+docker-compose -f $PWD/client/docker-compose.yml up --abort-on-container-exit
+docker-compose -f $PWD/client/docker-compose.yml down
+
+docker-compose -f $PWD/server/docker-compose.yml up --abort-on-container-exit
+docker-compose -f $PWD/server/docker-compose.yml down
```

Comment:
Instead of having a shell script that never runs in CI, it's best to have it integrated into pytest (with something like https://pypi.org/p/pytest-docker-compose / https://pypi.org/p/lovely-pytest-docker / or similar) and GHA.